script/server

```diff
@@ -87,7 +87,7 @@ if [ $? -eq 0 ]; then
 fi
 
 # Check if bundle is complete
-if ! bundle check > /dev/null
+if ! bin/bundle check > /dev/null
```

Comment:
Please open a separate pull request for that since this is not related at all to the changes you're proposing here.